File: NullgModels/PilotModels.py
from pydantic import Field

from NullgModels.NullGBaseModels import NullGBaseModel


class Skills(NullGBaseModel):
    piloting: int = Field(description="", default=5)
    gunnery: int = Field(description="", default=4)


class PilotData(NullGBaseModel):
    id: str = Field(description="", default=None)
    firstName: str = Field(description="", default=None)
    lastName: str = Field(description="", default=None)
    skills: Skills = Field(description="", default=None)
    bio: str = Field(description="", default=None)
    organizationId: str = Field(description="", default=None)
    kills: int = Field(description="", default=None)
    deaths: int = Field(description="", default=None)
    imageUrl: str = Field(description="", default=None)

File: SwiftModels/PilotModels.swift
import Foundation

public struct Skills: Codable, Equatable {
    public var piloting: Int? = 5
    public var gunnery: Int? = 4
}

public struct PilotData: Codable, Equatable {
    public var id: String?
    public var firstName: String?
    public var lastName: String?
    public var skills: Skills?
    public var bio: String?
    public var organizationId: String?
    public var kills: Int?
    public var deaths: Int?
    public var imageUrl: String?
}
